src/features/Header/Header.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+
+import CustomButton from '../../components/CustomButton/CustomButton';
+import ErrorMessage from '../../components/ErrorMessage/ErrorMessage';
+import { logOut } from '../../redux/userAccounts/userAccount.actions';
+import { isLoggedSelector, userDataSelector } from '../../redux/userAccounts/userAccount.selectors';
+import { errorMessageSelector } from '../../redux/movies/movies.selectors';
+import { propTypesShapes } from '../../constants';
+import styles from './Header.module.scss';
+
+const Header = ({ userData, isLogged, logOut, errorMessage }) => {
+  const logOutHandler = () => {
+    localStorage.removeItem('currentUser');
+
+    logOut();
+  };
+
+  return (
+    <header className={styles.header}>
+      { isLogged && (<NavLink className={styles.homeLink} to='/home'>Home</NavLink>) }
```

Comment:
no need to wrap in braces, `isLogged && <Nav.....`, also see 33 line, it's correct
And please pay more attention to make your code more consistent